assets/javascripts/comments.js

```diff
@@ -46,21 +46,21 @@ function updateNumerOfComments() {
   }
 }
 
-function deleteComment(deleteButton) {
-  const author = deleteButton.dataset.author;
-  if (!window.confirm('Do you really want to delete the comment written by ' + author + '?')) {
-    return;
-  }
-  $.ajax({
-    url: deleteButton.dataset.deleteUrl,
-    method: 'DELETE',
-    success: () => {
-      $(deleteButton).parents('.comment-row, .pinned-comment-row').remove();
-      updateNumerOfComments();
-    },
-    error: showXhrError.bind(undefined, "The comment couldn't be deleted: ")
-  });
-}
+//function deleteComment(deleteButton) {
+//  const author = deleteButton.dataset.author;
+//  if (!window.confirm('Do you really want to delete the comment written by ' + author + '?')) {
+//    return;
+//  }
+//  $.ajax({
+//    url: deleteButton.dataset.deleteUrl,
+//    method: 'DELETE',
+//    success: () => {
+//      $(deleteButton).parents('.comment-row, .pinned-comment-row').remove();
+//      updateNumerOfComments();
+//    },
+//    error: showXhrError.bind(undefined, "The comment couldn't be deleted: ")
+//  });
+//}
```

Comment:
Feel free to delete the code completely. This way the diff statistics tell use how many lines of custom code we avoided.